src/tools/android/java/com/google/devtools/build/android/ManifestMergerAction.java

```diff
@@ -174,8 +174,7 @@ private static Path removePermissions(Path manifest, Path outputDir)
       }
     }
     // Write resulting manifest to the output directory, maintaining full path to prevent collisions
```

Comment:
I think this comment is no longer valid with your change. The full path is no longer maintained; rather, a temporary file is created at `$outputDir/.xml/${gibberish}AndroidManifest`.